package.json

```diff
@@ -16,12 +16,13 @@
     "node": ">=18.19.0"
   },
   "scripts": {
+    "build:extended": "tsx scripts/build.ts extended-json",
     "build": "tsx scripts/build.ts package",
     "dist": "tsx scripts/dist.ts",
     "feature-init": "tsx scripts/feature-init.ts",
     "format": "npx prettier --write .",
     "schema:write": "npm run schema -- --out ./schemas/data.schema.json",
-    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --path ./types.ts",
+    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --path ./types.ts --type=WebFeaturesData",
```

Comment:
This bit is pretty serious:

I figured out what that top-level `$ref` was for. Without it, it was just a list of definitions without asserting anything using those definitions. Without this option, _any_ object will pass validation against our schema. 🙄

Reply:
Oh... let's put it back!